data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/S3SourceConfig.java

```diff
@@ -50,6 +50,9 @@ public class S3SourceConfig {
     @JsonProperty("records_to_accumulate")
     private int numberOfRecordsToAccumulate = DEFAULT_NUMBER_OF_RECORDS_TO_ACCUMULATE;
 
+    @JsonProperty("disable_bucket_ownership_validation")
+    private boolean disableBucketOwnershipValidation = false;
```

Comment:
I wanted to call out that this is validating the bucket ownership by default. Do we want to swap it?